samples/bootloader/src/main.c

```diff
@@ -154,25 +156,31 @@ static void boot_from(u32_t *address)
 	}
 
 	__DSB(); /* Force Memory Write before continuing */
-	__ISB(); /* Flush and refill pipeline with updated premissions */
+	__ISB(); /* Flush and refill pipeline with updated permissions */
 
-	VTOR = (u32_t)address;
+	VTOR = fw_info->firmware_address;
 
-	fw_abi_provide((u32_t)address);
+	fw_abi_provide(fw_info);
 
 	/* Set MSP to the new address and clear any information from PSP */
-	__set_MSP(address[0]);
+	__set_MSP(vector_table[0]);
 	__set_PSP(0);
 
 	/* Call reset handler. */
-	((void (*)(void))address[1])();
+	((void (*)(void))vector_table[1])();
 	CODE_UNREACHABLE;
 }
 
 void main(void)
 {
-	int err;
-	err = fprotect_area(PM_B0_ADDRESS, PM_B0_SIZE);
+	int err = fprotect_area(PM_B0_ADDRESS, PM_B0_SIZE);
+
+#if defined(CONFIG_SB_DEBUG_PORT_SEGGER_RTT)
```

Comment:
Why is this introduced? Wasn't this here already?

Reply:
This shouldn't be here :P. Removed